lib/auction.rb

```diff
@@ -0,0 +1,30 @@
+class Auction
```

Comment:
Great choice of enumerables for the methods in this class - I would have loved to see some tests though!